### DISCUSSION TITLE
TP-Link TL-WN823N v1 *working* monterey 12.7.2 MacBookAir7,2

### DISCUSSION THREAD

**Discussion by agomezalvic** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#326] [General]
Hi,
Just to say that v1 of this adapter is working (saw that v2 and v3 are in the list as reported to work but not v1).

Installed [Wireless USB Big Sur Adapter-V17](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/releases/tag/V17) on Monterey and connected to wifi router without problems.
Thanks !!!

**Comment by chris1111** [comment]
[DONE](https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/commit/c575c6f8d3b1c38b6d522e516b9e99178cf431c6) Thanks

**Comment by rlazar021982-eng** [comment]
the driver is not availabl
